src/main/java/com/mathworks/ci/tools/MatlabInstaller.java

```diff
@@ -170,6 +170,10 @@ private MatlabRelease parseRelease(String release) throws InstallationFailedExce
             }
         }
 
+        if (name.startsWith("r")) {
```

Comment:
Honestly probably don't need the if statement if we're sure that the release format will always be "R<year>", but I figured it can't hurt to verify.